src/suites/rulesets/lighthouse-api-standards.yaml

```diff
@@ -22,8 +22,48 @@ rules:
       field: security
       function: truthy
 
+  va-endpoint-default-errors-bad-request:
+    description: A response with error status code 400 is required for all endpoints.
+    given: $.paths..responses
+    severity: error
+    then:
+      field: '400'
+      function: truthy
+
+  va-endpoint-default-errors-unauthorized:
+    description: A response with error status code 401 is required for all endpoints.
+    given: $.paths..responses
+    severity: error
+    then:
+      field: '401'
+      function: truthy
+
+  va-endpoint-default-errors-forbidden:
+    description: A response with error status code 403 is required for all endpoints.
+    given: $.paths..responses
+    severity: error
+    then:
+      field: '403'
+      function: truthy
+
+  va-endpoint-default-errors-not-found:
+    description: A response with error status code 404 is required for all endpoints.
+    given: $.paths..responses
+    severity: error
+    then:
+      field: '404'
+      function: truthy
+
+  va-endpoint-default-errors-too-many-requests:
+    description: A response with error status code 429 is required for all endpoints.
+    given: $.paths..responses
+    severity: error
+    then:
+      field: '429'
+      function: truthy
```

Comment:
429 is a response we add to all endpoints since this is a gateway reply and not something the API Team is required to be coding for in addition to the gateway.  Maybe this has changed and you are now requiring this?  

I bring this up because I'm not sure when in the onboarding process the lighthouse-api-standards will be ran?  Will it be before the OAS is transformed by docserver?  Is is part of pre-qualifying an API to be on the public platform?

Reply:
We'll use it for both onboarding APIs and to audit existing APIs. If DocServer handles that, we can leave the 429 one out. When we had teams designing their APIs, we'd have them include it so that we/they could see what the final product would look like.

Reply:
I'll remove the 429 check since it's unnecessary, as the response is added later.